packages/cli/lib/projects.js

```diff
@@ -699,9 +699,24 @@ const makePollTaskStatusFunc = ({
 
     return new Promise((resolve, reject) => {
       const pollInterval = setInterval(async () => {
-        const taskStatus = await statusFn(accountId, taskName, taskId).catch(
-          reject
```

Comment:
So I think this was the problematic line.  Even if the `catch` block calls the `reject` methods the rest of the function will execute with a nullable `taskStatus`

Reply:
This is a little more complicated than most other cases in the repo due to the polling, but usually we try to surface errors from the backend using `logApiErrorInstance`. Probably worth giving that a shot and seeing if we at least get a helpful error. You could maybe try using `getAxiosErrorWithContext` from `local-dev-lib` and using that for the promise rejection.

If the backend error _isn't_ helpful, I like your solution!